.gitlab/olcf-gitlab-ci.yml

```diff
@@ -6,10 +6,18 @@ hipcc:
       aud: https://code.olcf.ornl.gov
   script:
     - module load rocm/6.0
-    - cmake -B build_hip -DCMAKE_CXX_COMPILER=hipcc -DKokkos_ENABLE_HIP=ON -DKokkos_ENABLE_TESTS=ON
-    - cmake --build build_hip -j48
-    - cd build_hip
-    - ctest -E Kokkos_CoreUnitTest_DeviceAndThreads -V
+    - export CMAKE_BUILD_PARALLEL_LEVEL=48
+    - export ENV_CMAKE_OPTIONS=""
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-D CMAKE_CXX_COMPILER=hipcc"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-D Kokkos_ENABLE_HIP=ON"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-D Kokkos_ENABLE_TESTS=ON"
+    - ctest -VV
+        -E Kokkos_CoreUnitTest_DeviceAndThreads
+        -D CDASH_MODEL="Nightly"
+        -D CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS}"
+        -D CTestRun.cmake
+        -D CTEST_SITE="frontier"
+        -D CTEST_BUILD_NAME="hipcc-rocm/6.0"
```

Comment:
I am not familiar with the YAML syntax for GitLab.
Please confirm that you validated it.  My understanding is that GitLab has some sort of CI lint tool to validate configuration files.

Reply:
![image](https://github.com/user-attachments/assets/bc8d9726-a111-4288-aa0c-673d8beabfb9)

Reply:
Thanks